operations/research-and-development/product/release-process/release-overview.md

```diff
@@ -89,10 +152,22 @@ Mattermost numbers stable releases in the following format:
 
   - A: See instructions here: https://developers.mattermost.com/internal/mobile-build-process/.
 
-**Q: How does updating dependencies work?**
-
-  - A: Dependency updates will only occur in feature releases, unless they contain security fixes.
-
 **Q: What is the process for community PRs?**
 
   - A: Review, merge, and cherry-pick.
+
+**Q: Will RC testing remain, or will RC testing process change?**
+
+  - There will be some manual RC testing until all release tests are automated.
+
+**Q: Are mobile and desktop included in the cloud first strategy?**
+  - Mobile and desktop apps are not affected at this point.
+
+**Q: Creating tickets for features that were cut from on-prem - Is this also how it continues to be tracked to see whether it's ready for the next release?**
+  - Any feature that's promoted out of the feature flag process would be turned on for the on-prem releases. That way on-prem only gets fully ready or tested features and not ones still being tested. There will be some cases where we need to make a change to fix or change something for the on-prem release (e.g. a bug that only happens if you use a certain on-prem setting or a certain database we don't use in cloud). In those cases the on-prem release might have some changes that are not a direct copy of the cloud build.
+
+**Q: What information does the customer support team need for Cloud releases?**
+  - The Cloud Releases channel in the Staff team is used for release updates and for posting the changelog. This may be automated in the future. We're also planning to have a dashboard that shows the current version deployed to cloud, and which commits are in.
```

Comment:
```suggestion
  - The Cloud Releases channel in the Staff team is used for release updates and for posting the changelog. This may be automated in the future. We're also planning to have a dashboard that shows the current version deployed to Cloud, and which commits are included.
```